Bugfix for version 0.0.1 (reported by Maniac@vdrportal):
Either make sure svdrpservice is loaded before remotetimers or apply
this patch.

--- timers.c.orig	2007-03-16 07:39:06.000000000 +0100
+++ timers.c	2007-03-16 07:41:27.000000000 +0100
@@ -48,7 +48,7 @@
 // cRemoteTimers -------------------------------------------------
 
 cRemoteTimers::cRemoteTimers(): refcount(0) {
-	plugin = cPluginManager::GetPlugin("svdrpservice");
+	plugin = NULL;
 	svdrp.handle = -1;
 }
 
@@ -59,6 +59,8 @@
 
 bool cRemoteTimers::Connect() {
 	refcount++;
+	if (!plugin)
+		plugin = cPluginManager::GetPlugin("svdrpservice");
 	if (plugin && svdrp.handle < 0) {
 		svdrp.serverIp = RemoteTimersSetup.serverIp;
 		svdrp.serverPort = RemoteTimersSetup.serverPort;
